tests/sentry/processing/realtime_metrics/test_redis.py

```diff
@@ -670,3 +671,49 @@ def test_get_durations_for_projects_with_gap(
         DurationHistogram(timestamp=140, histogram=empty_histogram()),
         DurationHistogram(timestamp=150, histogram=hist2),
     ]
+
+
+#
+# was_recently_moved()
+#
+
+
+def test_was_recently_moved_noop(store: RedisRealtimeMetricsStore) -> None:
+    store.remove_projects_from_lpq({42})
+    assert store.was_recently_moved(42)
+
+
+def test_was_recently_moved_added(store: RedisRealtimeMetricsStore) -> None:
+    store.add_project_to_lpq(42)
+    assert store.was_recently_moved(42)
+
+
+def test_was_recently_moved_removed(
+    store: RedisRealtimeMetricsStore, redis_cluster: redis._RedisCluster
+) -> None:
+    redis_cluster.sadd(LPQ_MEMBERS_KEY, 42)
+    store.remove_projects_from_lpq({42})
+    assert store.was_recently_moved(42)
+
+
+#
+# recently_moved_projects()
+#
+
+
+def test_recently_moved_projects_noop(store: RedisRealtimeMetricsStore) -> None:
+    store.remove_projects_from_lpq({42})
+    assert store.recently_moved_projects() == {42}
```

Comment:
maybe a bit related to your Q about mocking/fixtures:

do all of the tests get a completely empty redis store? As in: are they completely independent?
As in: is it possible that code is broken, but the project is part of "recently moved" by the side effect of another test?

In that case, might be good to use unique IDs for each test, and update the assertions accordingly.

Reply:
Another way to do it would be to change `store._prefix` to something unique in each test.

Reply:
Like this:
```python
def test_name() -> str:
    import sys
    return sys._getframe(1).f_code.co_name

def test_increment_project_event_counter_same_bucket(
    store: RedisRealtimeMetricsStore, redis_cluster: redis._RedisCluster
) -> None:
    store._prefix = test_name()
    store.increment_project_event_counter(17, 1147)
    store.increment_project_event_counter(17, 1149)

    assert redis_cluster.get(f"{test_name()}:counter:10:17:1140") == "2"